src/Game/GamePage.js

```diff
@@ -1,21 +1,37 @@
 import { useEffect, useMemo, useRef, useState } from "react";
 import { supabase } from "../supabase_client";
+import "./Game.css";
+import { sixSidedDice, fourSidedDice } from "../utils";
 
 const GamePage = (props) => {
   const { roomCode, playerName } = props;
   const [totalPlayerData, setTotalPlayerData] = useState([]);
   const [attackTarget, setAttackTarget] = useState("Select a target");
+  const [currentTurnPlayer, setCurrentTurnPlayer] = useState(0);
 
   const handleClick = async () => {
     const targetPlayer = totalPlayerData.find(
       (playerData) => playerData.name === attackTarget
     );
 
+    const sixDice = sixSidedDice();
+    const fourDice = fourSidedDice();
+    const diceDifference = sixDice - fourDice;
+
     if (targetPlayer !== undefined) {
-      const { data, error } = await supabase
-        .from("Players")
-        .update({ Hitpoints: targetPlayer.Hitpoints - 20 })
-        .eq("name", attackTarget);
+      if (diceDifference > 0) {
+        const { data, error } = await supabase
```

Comment:
```suggestion
        await supabase
```
We should stop assigning variables that aren't being used